logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -124,6 +126,14 @@ class UserDataSource(
             }
         }
 
+    override suspend fun fetchUsersIfUnknownByIds(ids: Set<UserId>): Either<CoreFailure, Unit> {
+        val qualifiedIDList = ids.map(idMapper::toDaoModel)
+        val knownUsers = userDAO.getUsersByQualifiedIDList(ids.map(idMapper::toDaoModel)).first()
```

Comment:
Can we create a new DAO function that will return the result as a list, not a flow?

Reply:
✅